use-cases/using-Intel-QAT-and-kata.md

```diff
@@ -0,0 +1,484 @@
+needs to be used then please refer to the link above on how to add it in.
+
+```sh
+$ mkdir -p $OSBUILDER
+$ cd $OSBUILDER
+$ git clone https://github.com/kata-containers/osbuilder.git
+$ export ROOTFS_DIR=${OSBUILDER}/osbuilder/rootfs-builder/rootfs
+$ export EXTRA_PKGS='kmod'
+
+# Make Sure that the kata agent version matches the installed kata-runtime version
+$ export AGENT_VERSION=$(kata-runtime version | head -n 1 | grep -o "[0-9.]\+")
+$ cd ${OSBUILDER}/osbuilder/rootfs-builder
+$ sudo rm -rf ${ROOTFS_DIR}
+$ script -fec 'sudo -E GOPATH=$GOPATH USE_DOCKER=true SECCOMP=no ./rootfs.sh clearlinux'
+```
+
+### Compile Intel QAT drivers for Kata Containers Kernel and add to Kata Containers rootfs
+
+After the Kata Containers kernel is built with the proper configuration flags 
+the Intel QAT drivers must be built against that Kata Containers kernel 
+version in much the same way they were built above for the host OS. You must 
+set the `KERNEL_SOURCE_ROOT` variable to the Kata Containers kernel source 
+directory and build the Intel QAT drivers again.
+
+```sh
+$ cd $GOPATH
+$ export LINUX_VER=$(ls -d kata*)
+$ export KERNEL_MAJOR_VERSION=$(awk '/^VERSION =/{print $NF}' $GOPATH/$LINUX_VER/Makefile)
+$ export KERNEL_PATHLEVEL=$(awk '/^PATCHLEVEL =/{print $NF}' $GOPATH/$LINUX_VER/Makefile)
+$ export KERNEL_SUBLEVEL=$(awk '/^SUBLEVEL =/{print $NF}' $GOPATH/$LINUX_VER/Makefile)
+$ export KERNEL_EXTRAVERSION=$(awk '/^EXTRAVERSION =/{print $NF}' $GOPATH/$LINUX_VER/Makefile)
+$ export KERNEL_ROOTFS_DIR=${KERNEL_MAJOR_VERSION}.${KERNEL_PATHLEVEL}.${KERNEL_SUBLEVEL}${KERNEL_EXTRAVERSION}
+
+$ cd $QAT_SRC
+$ sudo -E make clean
+
+$ KERNEL_SOURCE_ROOT=$GOPATH/$LINUX_VER ./configure --disable-qat-lkcf --enable-icp-sriov=guest
+$ sudo -E make quickassist-all -j$(nproc)
+$ sudo -E make INSTALL_MOD_PATH=$ROOTFS_DIR qat-driver-install -j$(nproc)
+
+# usdm_drv also needs to be copied into rootfs modules path and depmod run
+$ sudo cp $QAT_SRC/build/usdm_drv.ko $ROOTFS_DIR/usr/lib/modules/${KERNEL_ROOTFS_DIR}/updates/drivers  
+
+# Note: usdm_drv will be loaded by configuring kata kernel boot parameter in kata config
+# Ignore any errors on modules.builtin and modules.order
+$ sudo depmod -a -b ${ROOTFS_DIR} ${KERNEL_ROOTFS_DIR}
+
+# Create Kata rootfs
+$ cd ${OSBUILDER}/osbuilder/image-builder
+$ script -fec 'sudo -E USE_DOCKER=true ./image_builder.sh ${ROOTFS_DIR}'
+```
+
+### Copy Kata rootfs
+
+```sh
+$ mkdir -p $KATA_ROOTFS_LOCATION
+$ cp ${OSBUILDER}/osbuilder/image-builder/kata-containers.img $KATA_ROOTFS_LOCATION
+```
+
+### Update Kata configuration to point to custom kernel and rootfs
+
+You must update the `configuration.toml` for Kata Containers to point to the 
+custom kernel, custom rootfs, and to specify which modules to load when the 
+virtual machine is booted when a container is run. The example below assumes 
+an Intel QAT is installed and those modules need to be loaded.
```

Comment:
Lines 357-358 suggested rewrite: 
... The following example assumes you installed an Intel QAT and you need to load those modules.